crates/typst-library/src/compute/data.rs

```diff
@@ -105,6 +105,10 @@ impl From<Readable> for Bytes {
 /// Display: CSV
 /// Category: data-loading
 #[func]
+#[scope(
+    scope.define("decode", csv_decode_func());
+    scope
```

Comment:
no CSV encode?

Reply:
I've focused on serde functionality first; so if all that is ok; i will have a look at csv-encoding